opus_codec_registration.src.html

```diff
@@ -0,0 +1,110 @@
+Editor: Chris Cunningham, w3cid 114832, Google Inc. https://www.google.com/
+Editor: Paul Adenot, w3cid 62410, Mozilla https://www.mozilla.org/
+Editor: Bernard Aboba, w3cid 65611, Microsoft Corporation https://www.microsoft.com/
+
+Abstract: This registration is entered into the [[webcodecs-codec-registry]].
+    It describes, for Opus, the (1) fully qualified codec strings, (2)
+    the {{AudioDecoderConfig.description}} bytes, (3) the codec-specific
+    {{EncodedAudioChunk}} [=EncodedAudioChunk/[[internal data]]=] bytes, and (4)
+    the values of {{EncodedAudioChunk}} [=EncodedAudioChunk/[[type]]=].
+
+    The registration is not intended to include any information on whether a
+    codec format is encumbered by intellectual property claims. Implementers and
+    authors are advised to seek appropriate legal counsel in this matter if they
+    intend to implement or use a specific codec format. Implementers of
+    WebCodecs are not required to support the Opus codec.
+
+    This registration is non-normative.
+
+Markup Shorthands:css no, markdown yes, dfn yes
+!Participate: <a href="https://github.com/w3c/webcodecs">Git Repository.</a>
+!Participate: <a href="https://github.com/w3c/webcodecs/issues/new">File an issue.</a>
+!Version History: <a href="https://github.com/w3c/webcodecs/commits">https://github.com/w3c/webcodecs/commits</a>
+</pre>
+
+<pre class='anchors'>
+spec: WEBCODECS; urlPrefix: https://w3c.github.io/webcodecs/#
+    type: attribute
+        text: AudioDecoderConfig.description; url: dom-audiodecoderconfig-description
+    type: dfn
+        for: EncodedAudioChunkType; text: key; url: dom-encodedaudiochunktype-key
+        for: EncodedAudioChunk; text: [[internal data]]; url: dom-encodedaudiochunk-internal-data-slot
+        for: EncodedAudioChunk; text: [[type]]; url: dom-encodedaudiochunk-type-slot
+    type: method
+        for: AudioDecoder; text: AudioDecoder.decode; url: dom-audiodecoder-decode
+    type: interface
+        text: EncodedAudioChunk; url: encodedaudiochunk
+    type: dictionary
+        text: AudioDecoderConfig; url: dictdef-audiodecoderconfig
+</pre>
+
+<pre class='biblio'>
+{
+  "OPUS": {
+    "href": "https://datatracker.ietf.org/doc/html/rfc6716",
+    "title": "RFC 6716: Definition of the Opus Audio Codec",
+    "publisher": "IETF",
+    "date": "September 2012"
+  },
+  "OPUS-IN-OGG": {
+    "href": "https://datatracker.ietf.org/doc/html/rfc7845",
+    "title": "RFC 7845: Ogg Encapsulation for the Opus Audio Codec",
+    "publisher": "IETF",
+    "date": "April 2016"
+  }
+}
+</pre>
+
+Fully qualified codec strings {#fully-qualified-codec-strings}
+==============================================================
+
+The codec string is `"opus"`.
+
+EncodedAudioChunk data {#encodedaudiochunk-data}
+================================================
+
+An {{EncodedAudioChunk}} containing Opus can be in two different formats,
+depending on if {{AudioDecoderConfig.description}} has been set during
```

Comment:
Nit: I think it's better to say that chunks can be in one of two formats, and leave the association with `description` for the next section. In this case we don't need to also map an enum value so I don't think that will make it as indirect as the AVC registration.